src/main/kotlin/com/team4099/robot2022/subsystems/Drivetrain.kt

```diff
@@ -35,42 +34,26 @@ object Drivetrain : SubsystemBase() {
   val wheels =
       listOf(
           Wheel(
-              CANSparkMax(
-                  Constants.Drivetrain.FRONT_LEFT_DIRECTION_ID,
-                  CANSparkMaxLowLevel.MotorType.kBrushless),
-              CANSparkMax(
-                  Constants.Drivetrain.FRONT_LEFT_SPEED_ID,
-                  CANSparkMaxLowLevel.MotorType.kBrushless),
+              TalonFX(Constants.Drivetrain.FRONT_LEFT_DIRECTION_ID),
```

Comment:
I would take this opportunity to rename stuff here to make more sense (can we go with steering instead of direction and drive instead of speed?)